projectq/ops/_gates_test.py

```diff
@@ -80,6 +80,14 @@ def test_t_gate():
     assert isinstance(_gates.Tdagger, type(get_inverse(gate)))
 
 
+def test_sqrtx_gate():
+    gate = _gates.SqrtXGate()
+    assert str(gate) == "SqrtX"
+    assert np.array_equal(gate.matrix * gate.matrix,
```

Comment:
This is a useful test 👍 
Can you also add a trivial test checking the gate matrix? As there are always two different roots, we might want to make sure we don't change convention of which one we pick